.circleci/scripts/auto-rebase-branches.txt

```diff
@@ -1,2 +1,3 @@
+jz/rebase-custom1
```

Comment:
Is this just a testing branch?

Reply:
Rebasing doesn't work for the other 2 merchant branches due to code conflicts so I added this one. The ci job just do rebase and tests for the first in the list. @njfaries

Reply:
I see, makes sense.